document-store/src/main/java/org/hypertrace/core/documentstore/mongo/query/parser/filter/MongoContainsRelationalFilterParser.java

```diff
@@ -0,0 +1,26 @@
+package org.hypertrace.core.documentstore.mongo.query.parser.filter;
+
+import com.mongodb.BasicDBObject;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import org.hypertrace.core.documentstore.expression.impl.RelationalExpression;
+import org.hypertrace.core.documentstore.mongo.query.parser.filter.MongoRelationalFilterParserFactory.MongoRelationalFilterContext;
+
+@AllArgsConstructor
+public class MongoContainsRelationalFilterParser implements MongoRelationalFilterParser {
```

Comment:
`MongoContainsRelationalFilterParser` -> `MongoEqualRelationalFilterParser` or `MongoElementMatchRelationalFilterParser` ?

Reply:
Ignore this one, next I saw that we have enum for `CONTAINS` and so.